src/arch/xtensa/smp/init.c

```diff
@@ -44,7 +44,7 @@
 #include <sof/init.h>
 #include <sof/lock.h>
 #include <sof/notifier.h>
-#include <sof/schedule.h>
+#include <sof/edf_schedule.h>
```

Comment:
I think we should use generic schedule.h here.

Reply:
ack, we use schedule.h, the APIs will also be the same regardless of scheduler type. The scheduler type can be selected at runtime (by the taks) and also at buildtime (to reduce footprint if not used).

Reply:
Done